src/nxt_unit.h

```diff
@@ -291,31 +332,89 @@ int nxt_unit_response_is_websocket(nxt_unit_request_info_t *req);
 
 nxt_unit_request_info_t *nxt_unit_get_request_info_from_data(void *data);
 
+/*
+ * Send and deallocate a response data chunk.  The data is immediately sent to
+ * the client as a chunk using the "Transfer-Encoding: chunked" method.
+ *
+ * If the initial response was not yet sent with nxt_unit_response_send(), this
+ * function will automatically call it.
+ */
 int nxt_unit_buf_send(nxt_unit_buf_t *buf);
 
+/*
+ * Deallocate a response data chunk without sending it.
+ */
 void nxt_unit_buf_free(nxt_unit_buf_t *buf);
 
 nxt_unit_buf_t *nxt_unit_buf_next(nxt_unit_buf_t *buf);
 
+/*
+ * The maximum size that can be requested with nxt_unit_response_buf_alloc().
+ */
 uint32_t nxt_unit_buf_max(void);
 
+/*
+ * The minimum size that will be allocated by nxt_unit_response_buf_alloc().
+ */
 uint32_t nxt_unit_buf_min(void);
 
+/*
+ * Schedule a response to be sent to the client.  This will repeatedly call
```

Comment:
Does nxt_unit_response_write_nb still get repeatedly called? It appears to me that it is called once... Unless nxt_unit_response_write is being repeatedly called in which case we should clarify that in this doc string.